website/homepage/src/docs/obj_doc_template.html

```diff
@@ -48,6 +48,9 @@
   {% if is_component %}
   <p class="mb-2 text-lg text-gray-500">As input, {{ obj["tags"]["preprocessing"] }}</p>
   <p class="mb-2 text-lg text-gray-500">As output, {{ obj["tags"]["postprocessing"] }}</p>
+  {% if "examples" in obj["tags"]  %}
+  <p class="text-lg text-gray-500">Format expected for examples: <em>{{ obj["tags"]["examples"] }}</em></p>
```

Comment:
If it's missing the `Examples` tag in the docstring, it wont add a line about the examples.

![image](https://user-images.githubusercontent.com/41651716/178364341-72307b05-1e3c-4665-b89c-286ba21876f0.png)